admin-notice/admin-notice.php

```diff
@@ -26,3 +26,19 @@ function() use ( $admin_notice_controller ) {
 		do_action( 'vip_admin_notice_init', $admin_notice_controller );
 	}
 );
+
+add_action(
+	'vip_admin_notice_init',
+	function( $admin_notice_controller ) {
+		$admin_notice_controller->add(
+			new Admin_Notice(
+				'WordPress 5.9 is scheduled to be released on Tuesday, January 25th',
```

Comment:
Should we include something how this doesn't apply for pinned sites to avoid any panic?